### DISCUSSION TITLE
[Bug]: How to add imageData from other images to PencilBrush

### DISCUSSION THREAD

**Discussion by fangyinghua** [fabricjs/fabric.js#8845] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

5.2.1

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

16.18.1

### Link To Reproduction

null

### Steps To Reproduce

1. Add a black and white mask layer
2. Filtered out white data
3. How to create this data as points for PencilBrush

### Expected Behavior

 How to add imageData from other images to PencilBrush

### Actual Behavior

 How to add imageData from other images to PencilBrush

**Comment by ShaMan123** [comment]
Are you after the PatternBrush?
